alerts.go

```diff
@@ -4,6 +4,7 @@ import (
 	"fmt"
 )
 
+// Alert defines a policy for sending alarms to services when conditions are met
 type Alert struct {
```

Comment:
Should this be renamed to `AlertResponse` for full disambiguation?

Reply:
I think that's something we could do lib-wide before `1.0.0`, but not at this point.